front/app/containers/Admin/formBuilder/containers/projects/forms/edit/index.tsx

```diff
@@ -231,6 +197,10 @@ export const FormEdit = ({
     selectedField?.input_type !== 'page' || isPageDeletable
   );
 
+  const handleDragEnd = (result, nestedPageData: PageStructure[]) => {
+    reorderFields(result, nestedPageData, replace);
+  };
```

Comment:
As stated in my comment - we just need to update the selected field here or in the edit/utils somewhere as well if one is selected while dragging :+1: